regression/cbmc-incr-smt2/nondeterministic-int-assert/stdbool_example.c

```diff
@@ -0,0 +1,10 @@
+#include <stdbool.h>
+
+int main()
+{
+  int x, y;
+  bool equal = x == y;
+  __CPROVER_assert(equal, "Assert of integer equality.");
+  __CPROVER_assert(!equal, "Assert of not integer equality.");
```

Comment:
⛏️ What does `not integer equality` mean? Do you mean `inequality` in this context?

Reply:
I mean the not operator is being applied to the result of testing for integer equality. This is equivalent to testing for inequality with the `!=` operator, so the distinction is not really important here. The main purpose of this assertion is to ensure that the output traces include instances where equal is assigned a true value and equal is assigned a false value.